gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -605,7 +605,7 @@ private int addSharedSteps(List<CopyEntity> copyEntities, String fileSet, int in
 
     DeleteFileCommitStep deleteStep = DeleteFileCommitStep.fromPaths(targetFs, diffPathSet.pathsToDelete,
         this.dataset.properties);
-    copyEntities.add(new PrePublishStep(fileSet, Maps.<String, Object>newHashMap(), deleteStep, stepPriority++));
+    copyEntities.add(new PrePublishStep(fileSet, Maps.<String, Object>newHashMap(), deleteStep, stepPriority));
```

Comment:
It throws a http://findbugs.sourceforge.net/bugDescriptions.html#DLS_DEAD_LOCAL_STORE because `stepPriority` is a local variable, and `stepPriority++` will file pass the original value of `stepPriority` to the `PrePublishStep` constructor, and then increment the value of `stepPriority`. So the incremented value of `stepPriority` is never used.

I wasn't sure what the desired behavior was. By removing the `++` it basically does what it always did, but I can add a `+ 1` instead if that is the desired behavior.